bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -67,6 +71,32 @@ spec:
             source /env/_master
             set +o allexport
           fi
+          
+          # Perform DNS resolution against the DNS resolver first and /etc/hosts second
+          sed -i 's/hosts:      files dns myhostname/hosts:      dns files myhostname/g' /etc/nsswitch.conf
+
+          function watch_core_dns_ep () {
```

Comment:
maybe this could be a helper script installed by ovn-kubernetes rather than being duplicated 3 times...

Reply:
Indeed my first thought as well, I am guilty of laziness and didn't go the extra mile to implement helper scripts which are packaged in our images. Will do however